var/spack/repos/builtin/packages/rmgdft/package.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Rmgdft(CMakePackage):
+    """RMGDFT is a high performance real-space density functional code
+       designed for large scale electronic structure calculations."""
+
+    homepage = "http://www.rmgdft.org/"
+    git      = "https://github.com/RMGDFT/rmgdft.git"
+    maintainers = ['elbriggs']
+    tags = ['ecp', 'ecp-apps']
+    version('master', branch='master')
+    version('4.2.1', tag='v4.2.1')
```

Comment:
Will any users desire versions prior to `v4.2.1`?

Reply:
@naromero77 if users need an earlier version later, it's fine to add that later. It's too much to try to keep every version list exhaustive.